python/cugraph/tests/test_woverlap.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+from itertools import product
+import time
+
+import pytest
+
+import cudf
+import cugraph
+from cugraph.tests import utils
+from librmm_cffi import librmm as rmm
+from librmm_cffi import librmm_config as rmm_cfg
+import numpy as np
+
+
+def cugraph_call(cu_M, first, second):
+    # Device data
+    sources = cu_M['0']
+    destinations = cu_M['1']
+    weights_arr = cudf.Series(np.ones(max(sources.max(),
```

Comment:
At some point, we may want to actually have weights other than `1` to test this. Approving since this is consistent with how Jaccard is tested.